actors/miner/src/lib.rs

```diff
@@ -5513,15 +5581,28 @@ fn activate_sectors_deals(
         .activations
         .iter()
         .zip(claim_res.sector_claims)
-        .map(|(sector_deals, sector_claim)| DataActivationOutput {
-            unverified_space: sector_deals.nonverified_deal_space.clone(),
-            verified_space: sector_claim.claimed_space,
-            unsealed_cid: sector_deals.unsealed_cid,
+        .map(|(sector_deals, sector_claim)| {
+            let mut piece_infos_for_sector: Vec<(Cid, u64)> = vec![];
+
+            for verified_info in &sector_deals.verified_infos {
+                piece_infos_for_sector.push((verified_info.data, verified_info.size.0));
+            }
+
+            for unverified_info in &sector_deals.unverified_infos {
+                piece_infos_for_sector.push((unverified_info.data, unverified_info.size.0));
+            }
+            piece_infos.push(piece_infos_for_sector);
```

Comment:
This roundtrip to the market actor and returning the pieces in two lists, verified and unverified, means we've lost the ordering of pieces within the sector. It's not obvious how much of a problem that might, be but we should be able to retain it. We'll need to modify the market actor return value to maintain the order of the deals.